### ISSUE TITLE
nshield-may.py

### ISSUE THREAD

**Issue by oviliz** [fnzv/nShield#2]
The Quickstart says to run as a standalone software using:
`python nshield-may.py -dry`

I believe you mean:
`python nshield-main.py -dry`

In which case I get this:
```
Traceback (most recent call last):
  File "nshield-main.py", line 19, in <module>
    dryrun = int(config.get("conf","dryrun"))
  File "/usr/lib/python2.7/ConfigParser.py", line 607, in get
    raise NoSectionError(section)
ConfigParser.NoSectionError: No section: 'conf'
```

**Comment by fnzv** [comment]
Thank you for this issue, i fixed the typo on the README.
I assume you just run git clone of the repo and then executed `python nshield-main.py -dry` .
To make it work with dry run you must have a configuration file set so if it's not a production server you can run the install.sh script.

TLDR: if its a prod server --> copy & paste the example configuration file to /etc/nshield/nshield.conf
else --> run the install.sh (it will automatically install nginx, copy cfg files, logrotate & set cron)

Let me know if you need more help to set it up and running

**Comment by oviliz** [comment]
Thanks @fnzv .

It is a production server indeed and I must have used git clone. Being a production server I don't really want the software to install nginx as I have that already installed in front of Apache as ServerPilot.io does.

I may install that on my Linux laptop and see how it configures nginx and eventually replicate that on the production side.
